basics/pdf.php

```diff
@@ -0,0 +1,44 @@
+<?php
+// Include FPDF library
+require '../vendor/setasign/fpdf/fpdf.php';
+
+class PDF extends FPDF {
```

Comment:
Class name and Class file name should be exactly the same, i.e. same letter case.